src/staticanalysis/bot/tests/conftest.py

```diff
@@ -441,3 +441,16 @@ def mock_coverity_output():
     Load a real case clang output
     '''
     return os.path.join(MOCK_DIR, 'coverity.json')
+
+
+@pytest.fixture
+@responses.activate
+def mock_coverage():
```

Comment:
For some reason this doesn't work and I had to copy its content to the test directly...

Reply:
I had issues with the order of decorators when using pytest + responses